src/plugins/unified_search/public/query_string_input/text_based_languages_editor/resizable_button.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+
+import './resizable_button.scss';
+
+export function ResizableButton({
+  onMouseDownResizeHandler,
+}: {
+  onMouseDownResizeHandler: (mouseDownEvent: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+}) {
+  const setFocus = (e: React.MouseEvent<HTMLDivElement>) => e.currentTarget.focus();
+
+  return (
+    <div
```

Comment:
Should this be a `button` element instead (for better semantics and accessibility)? Also, would it also be possible to support resizing via up/down on the keyboard when focused?